### PR TITLE
Maint edit 23 3

### PR DESCRIPTION
**Pull request by mhodki**
For qualified person
1. Added line to restriction as suggested by Milos email 22/3 to Melinda
2. Changed class to primitive
3. No change to FOL definition as it is the intended definition
4. Note that the FOL description provided is MORE expressive than the OWL restrictions. If we made it a defined class suggested by Milos we ran the risk that a (qualified) welder doing an electrical task would be assumed to be a qualified electrical person.

For maintenance state
1. There is a label change for object property assoc with maintenance state

### CODE REVIEW

File: maintenance/MaintenanceReferenceOntology.rdf [hunk 1]
```diff
@@ -465,6 +484,7 @@
 	
 	<owl:ObjectProperty rdf:about="&iof-maint;stateOf">
 		<rdfs:subPropertyOf rdf:resource="&obo;BFO_0000167"/>
+		<rdfs:label>maintainableMaterialStateOf</rdfs:label>
```
Comment (mereolog) [inline maintenance/MaintenanceReferenceOntology.rdf:487]: Labels should be readable by humans, so in this case "maintainable material state of" might be better.

**Comment by mhodki** [comment]
Pawel, I've made the edit to the label and repushed.

File: maintenance/MaintenanceReferenceOntology.rdf [hunk 2]
```diff
@@ -439,6 +457,7 @@
 	
 	<owl:ObjectProperty rdf:about="&iof-maint;hasState">
 		<rdfs:subPropertyOf rdf:resource="&obo;BFO_0000056"/>
+		<rdfs:label>hasMaintainableMaterialState</rdfs:label>
```
[the remark this answers is not in the document]
Comment (Caitlin2694) [reply maintenance/MaintenanceReferenceOntology.rdf]: this has now been resolved

**Review by Caitlin2694 — APPROVED** [review]
I approve these changes after a minor edit to a label

**Review by mereolog — APPROVED** [review]

**Pull request merged**